From ebcdd9f658b2363dc962d7791d09e810497c4b2a Mon Sep 17 00:00:00 2001
From: Chirayu Desai <chirayudesai1@gmail.com>
Date: Tue, 25 Jun 2019 01:05:05 +0530
Subject: [PATCH] Tethering: Use QUAD 9 DNS instead of Google DNS

Change-Id: I1394b33e883d82c4d593d0f84bb0beb551cccc92
(cherry picked from commit cf7af52ef68c5d90872ee42083d308c5c5b790ad)
Signed-off-by: chiru2000 <chiranth@m.ms.evolution-x.org>
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
[minor adaptions to use QUAD 9 instead of CLOUDFLARE]
Signed-off-by: bananafunction <terathree2@web.de>
---
 .../android/networkstack/tethering/TetheringConfiguration.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Tethering/src/com/android/networkstack/tethering/TetheringConfiguration.java b/Tethering/src/com/android/networkstack/tethering/TetheringConfiguration.java
index 727317c555..0ba43af9b7 100644
--- a/Tethering/src/com/android/networkstack/tethering/TetheringConfiguration.java
+++ b/Tethering/src/com/android/networkstack/tethering/TetheringConfiguration.java
@@ -73,7 +73,7 @@ public class TetheringConfiguration {
         "192.168.48.2", "192.168.48.254", "192.168.49.2", "192.168.49.254",
     };
 
-    private static final String[] DEFAULT_IPV4_DNS = {"8.8.4.4", "8.8.8.8"};
+    private static final String[] DEFAULT_IPV4_DNS = {"149.112.112.112", "9.9.9.9"};
 
     @VisibleForTesting
     public static final int TETHER_USB_RNDIS_FUNCTION = 0;

